packages/airnode-node/src/api/processing.test.ts

```diff
@@ -1,59 +1,83 @@
 import { postProcessApiSpecifications, preProcessApiSpecifications } from './processing';
 import * as fixtures from '../../test/fixtures';
 
-describe('processing', () => {
```

Comment:
This file looks like it has a large diff because I removed the unnecessary upper level `describe('processing', () => {`. The only actual change is the addition of the test `it('makes reserved parameters inaccessible for HTTP gateway requests', async () => {`.